velox/dwio/parquet/reader/ParquetReader.cpp

```diff
@@ -1107,6 +1112,7 @@ class ParquetRowReader::Impl {
   const thrift::RowGroup* currentRowGroupPtr_{nullptr};
   uint64_t rowsInCurrentRowGroup_;
   uint64_t currentRowInGroup_;
+  uint32_t skippedStrides_{0};
```

Comment:
nit: shall we use 64-bit type as below?
https://github.com/facebookincubator/velox/blob/0b8c32298a879c748d5df43ca1407108ab7a17e5/velox/dwio/common/Statistics.h#L549